.editorconfig

```diff
@@ -9,19 +9,14 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-# Tabs in JS unless otherwise specified
 [**.js]
 indent_style = tab
 indent_size = 4
 
-[**.jsx]
+[**.ts]
```

Comment:
Do we need .tsx here as well?

Reply:
Nope. I don't foresee any .jsx or .tsx files in this project. Should just be plain .ts/.js.